tasks/download.yml

```diff
@@ -27,4 +38,7 @@
     dest: "{{ odoo_role_odoo_path }}"
     version: "{{ odoo_role_odoo_git_ref }}"
     depth: 1
-  when: odoo_role_odoo_edition == "oca"
+    force: yes
```

Comment:
this is to pull updates even if there are local changes. Otherwise Ansible crashes. We don't want local changes in deployed servers running code from git repositories.